astropy/coordinates/tests/test_regression.py

```diff
@@ -225,3 +226,30 @@ def test_regression_4996():
 
     # this is intentionally not allclose - they should be *exactly* the same
     assert np.all(suncoo.ra.ravel() == suncoo2.ra.ravel())
+
+
+def test_regression_4293():
+    """Really just an extra test on FK4 no e, after finding that the units
+    were not always taken correctly.  This test is against explicityly doing
+    the transformations on pp170 of Explanatory Supplement to the Astronomical
+    Almanac (Seidelmann, 2005).
+
+    See https://github.com/astropy/astropy/pull/4293#issuecomment-234973086
+    """
+
+    fk4 = FK4(10.00039633*u.deg, 40.99492542*u.deg)  # M31
+
+    Dc = -0.065838*u.arcsec
+    Dd = 0.35299*u.arcsec
+    # obliquity of Earth's orbit, B1950
+    o = 0.40920613517217536*u.rad
+
+    fk4noe_dec = (fk4.dec - (Dd*np.cos(fk4.ra) -
+                            Dc*np.sin(fk4.ra))*np.sin(fk4.dec) -
+                  Dc*np.tan(o)*np.cos(fk4.dec))
+    fk4noe_ra = fk4.ra - (Dc*np.cos(fk4.ra) +
+                          Dd*np.sin(fk4.ra)) / (np.cos(fk4.dec))
+
+    fk4noe = fk4.transform_to(FK4NoETerms)
+    assert_quantity_allclose(fk4noe.ra, fk4noe_ra, rtol=5e-7)
```

Comment:
Is this `rtol` value a "Magic Number," or is it set by some heuristic? Maybe just make an inline comment note about where that came from